### DISCUSSION TITLE
i want to add some hw to the board. and i have no clues about the pins(pwm fan / TOF Sensor)

### DISCUSSION THREAD

**Discussion by DvnGBulletZz** [witnessmenow/ESP32-Cheap-Yellow-Display#118] [Q&A]
hi

I'm looking to hook up a pwm fan and a tof sensor on this board but i have 0 clue if there are enough ports or if i have to desolder something. so if someone knows it would be amazing if i can get an anwser.

F.Y.I im not the best when it comes to looking at boards 
thanks guys <3

**Comment by witnessmenow** [comment]
There is info on the pins available here: https://github.com/witnessmenow/ESP32-Cheap-Yellow-Display/blob/main/PINS.md
